Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.py

```diff
@@ -0,0 +1,896 @@
+              'tags': argToList(args.get('tags')), 'internet-access': argToBoolean(args.get('internet_access', True)),
+              'archive-no-unpack': argToBoolean(args.get('archive_no_unpack', False)),
+              'ssl-inspection': argToBoolean(args.get('ssl_inspection', False)),
+              'localized-internet-country': args.get('localized_internet_country'),
+              'internet-simulation': argToBoolean(args.get('internet_simulation', False)),
+              'hybrid-code-analysis': argToBoolean(args.get('hybrid_code_analysis', True)),
+              'hybrid-decompilation': argToBoolean(args.get('hybrid_decompilation', False)),
+              'vba-instrumentation': argToBoolean(args.get('vba_instrumentation', True)),
+              'js-instrumentation': argToBoolean(args.get('js_instrumentation', True)),
+              'java-jar-tracing': argToBoolean(args.get('java_jar_tracing', True)),
+              'dotnet-tracing': argToBoolean(args.get('dotnet_tracing', True)),
+              'amsi-unpacking': argToBoolean(args.get('amsi_unpacking', True)),
+              'fast-mode': argToBoolean(args.get('fast_mode', False)),
+              'secondary-results': argToBoolean(args.get('secondary_results', False)),
+              'report-cache': argToBoolean(args.get('report_cache', False)),
+              'command-line-argument': args.get('command_line_argument'),
+              'live-interaction': argToBoolean(args.get('live_interaction', False)),
+              'document-password': args.get('document_password'), 'archive-password': args.get('archive_password'),
+              'start-as-normal-user': argToBoolean(args.get('start_as_normal_user', False)),
+              'language-and-locale': args.get('language_and_locale'),
+              'delete-after-days': arg_to_number(args.get('delete_after_days', 30)),
+              'encrypt-with-password': args.get('encrypt_with_password'),
+              'export-to-jbxview': argToBoolean(args.get('export_to_jbxview', False)),
+              'email-notification': argToBoolean(args.get('email_notification', False))}
+
+    return params
+
+
+def file_submission(client: Client, args: Dict[str, Any], params: Dict[str, Any], file: str,
+                    exe_metrics: ExecutionMetrics) -> PollResult:
+    """
+        Helper function that submits a file to Joe Sandbox.
+
+         Args:
+            client (Client): The client class.
+            args: (Dict[str, Any]): The command arguments.
+            params: (Dict[str, Any]): The submission parameters.
+            file: (str): The file to submit.
+            exe_metrics
+
+         Returns:
+             result: (PollResult): The parsed PollResult object.
+     """
+    file_path = demisto.getFilePath(file)
+    with open(file_path['path'], 'rb') as f:
+        res = client.submit_sample(sample=f, params=params, cookbook=args.get('cookbook'))
+        update_metrics(res, exe_metrics)
```

Comment:
I think here you can just raise success by 1, use update_metrics only for exceptions